test/module/app/getGeneralSettings.test.js

```diff
@@ -3,106 +3,1221 @@
  * test app module
  */
 const nock = require('nock');
-
-const common = require('../../utils/common');
-
-const KintoneAPIException = require('../../../src/exception/KintoneAPIException');
-const Connection = require('../../../src/connection/Connection');
-const Auth = require('../../../src/authentication/Auth');
-const App = require('../../../src/module/app/App');
+const common = require('../utils/common');
+const {App, Auth, Connection, KintoneAPIException} = require('../../../src/main.js');
 
 const URI = 'https://' + common.DOMAIN;
+const preLiveRoute = '/k/v1/preview/app/settings.json';
+const liveRoute = '/k/v1/app/settings.json';
+const guest_PreLiveRoute = `/k/guest/${common.GUEST_SPACEID}/v1/preview/app/settings.json`;
+const guest_LiveRoute = `/k/guest/${common.GUEST_SPACEID}/v1/app/settings.json`;
 
 const auth = new Auth();
 auth.setPasswordAuth(common.USERNAME, common.PASSWORD);
-
 const conn = new Connection(common.DOMAIN, auth);
-
 const appModule = new App(conn);
 
+const auth_API = new Auth();
```

Comment:
@vuong-le 
Please using camel style for defined variables.
Ex:
const authApi = new Auth();
const connGuestSpace = new Connection(common.DOMAIN, auth, common.GUEST_SPACEID);